release/mixer.sh

```diff
@@ -214,21 +214,24 @@ stage Mixer
 pushd "${MIXER_DIR}" > /dev/null
 
 section "Bootstrapping Mix Workspace"
-mixer_cmd init --upstream-url "${CLR_PUBLIC_DL_URL}" --upstream-version "${CLR_LATEST}"
-mixer_cmd config set Swupd.CONTENTURL "${DISTRO_URL}/update"
-mixer_cmd config set Swupd.VERSIONURL "${DISTRO_URL}/update"
-
 log_line "Looking for previous releases:"
-if [[ -z ${DS_LATEST} ]]; then
+if [[ -z "${DS_LATEST}" ]]; then
     log_line "None found. This will be the first Mix!" 1
     DS_UP_FORMAT=${CLR_FORMAT}
     # shellcheck disable=SC2034
     DS_UP_VERSION=${CLR_LATEST}
 
     var_save DS_UP_FORMAT
     var_save DS_UP_VERSION
+
+    log_line "Initializing Mixer Workspace"
+    mixer_cmd init --upstream-url "${CLR_PUBLIC_DL_URL}" --upstream-version "${CLR_LATEST}"
```

Comment:
I just checked with one of the mixer leads, and this looks like a correct change.  I think the current BKM is to run `mixer init` only once.